presentation/src/main/java/com/lighthouse/presentation/ui/main/MainActivity.kt

```diff
@@ -33,6 +35,18 @@ class MainActivity : AppCompatActivity() {
 
     private lateinit var binding: ActivityMainBinding
     private val viewModel: MainViewModel by viewModels()
+    private val callback = object : OnBackPressedCallback(true) {
+        override fun handleOnBackPressed() {
+            when (supportFragmentManager.fragments.filter { it.isVisible }[0]) {
+                is HomeFragmentContainer -> finish()
+                is SettingSecurityFragment -> {} // SettingSecurityFragment 에서 관리하는 부분을 옮기려고 했는데 안 되네요...
+                else -> {
+                    binding.bnv.selectedItemId = R.id.menu_home
+                    viewModel.gotoHome()
+                }
+            }
+        }
+    }
```

Comment:
{ it.isVisible }[0])이 무엇을 의미하는건가요?

Reply:
그런 용도라면 이런 코드가 더 적합해보여요!
```suggestion
supportFragmentManager.fragments.first { it.isVisible }
```

Reply:
적용완료